Added German localization file deDE.lua

meandmykeks opened the pull request.

meandmykeks committed:
Added German localization file deDE.lua

meandmykeks committed:
Update SoulshapeJournal.toc

christopheml commented:
Hello, thank you so much for providing a translation.
I'll assign it to the 0.9 milestone.
0.8 will see updates in translation strings, I'll notify you in the PR for the appropriate updates.

christopheml commented:
Hello, there are a few translation key changes upcoming today for the 0.9 version, I'll start integrating translations after you have a chance to update the PR.

meandmykeks committed:
Merge branch 'christopheml:master' into master

meandmykeks committed:
updated keys for 0.9

Added：
　- "Covenant Callings reward chests"
　- "Normal" (dungeon difficulty)
　- "Mythic" (dungeon difficulty
　- "Goat Soul Guide"


Removed：
　- "Normal or Heroic" (dungeon difficulty)

meandmykeks committed:
Merge branch 'christopheml:master' into master

meandmykeks committed:
added Crittershape key for 0.9

christopheml committed:
Merge branch 'master' into master
